docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+:::
+## Resources
+
+::::{dropdown} 1. What are the time limits on the `lowpriority` partition?
+Partition limits for the `lowpriority` partition are the same as the default partition (`short`), which has a 4h default walltime and a 24h max-walltime.
+::::
+
+::::{dropdown} 2. What are the resource and job limits on the `lowpriority` partition?
+Currently, the resource limits and the number for CPU-supported low priority jobs on the `lowpriority` partition are the same as the default partition (see {ref}`partition-names`). The resource limits and the number for GPU-supported low priority jobs on the `lowpriority` partition are the same as the gpu partition (see {ref}`partition-names`).
+::::
+
+::::{dropdown} 3. Is there a limit on the number of CPUs or GPUs that can be requested when trying to use the `lowpriority` partition?
+
+The maximum limit for number of CPUs, GPUs, job time etc. will be the same as that on the `short` and `gpu` partitions. Please check our partitions (see {ref}`partition-names`) page for the core and RAM limits on these partition. 
+::::
+
+::::{dropdown} 4. I am an owner of a private partition, will this affect my group’s job wait time?
+
+As a PI who purchased your own equipment, jobs submitted by the members of your private partition to your own resources always have the highest priority. When submitting jobs to your private partition, any low priority jobs currently running on your servers will be killed and sent back to the queue, introducing a slight delay (~30s) on the first job submitted. This new service is designed to favor partition owners heavily.
+::::
+
+::::{dropdown} 5. Will the `lowpriority` partition affect the limits on my own server(s)?
+The `lowpriority` partition will not affect current partition definitions — it is just an additional partition. As is true now, PIs will be able to request their private partitions be configured according to their research requirements. 
+::::
+
+::::{dropdown} 6. Will the software that my group uses exclusively on our server(s) be available on other servers?
+
+Yes, all software used in the private partitions is available on all servers. Software that is restricted to a particular group’s license will remain restricted to members of that group, but they will be able to use that software on any server. 
+::::
+
+::::{dropdown} 7. What hardware will be part of the `lowpriority` partition?
+All PI-owned hardware purchased on or after 2019 will be part of the `lowpriority` partition.
+::::
+
+::::{dropdown} 8. Do you have a clear inventory of the various resources available through the `lowpriority` partition?
+
+Throughout the testing phase, RC will be updating its `low priority` documentation (see {ref}`low-priority`)to reflect the resources that are part of public and private partitions. Please check the documentation (see {ref}`low-priority`) regularly for latest information. See our CPUs ({ref}`hardware-overview`) and GPUs ({ref}`working-gpus`) pages.
```

Comment:
This answer is really choppy and confusing. Is this question actually necessary, or can we just remove it completely? It just links to a bunch of pages we already link to in other areas and commits us to updating these pages frequently. If we do need to keep it we need to clean up the links.

Reply:
It might be nice to keep just for the GPUs to be honest. The CPUs I don't think would be as important but some ML/DL workflows require certain memory requirements and it would be beneficial for them to know what is available in the partition. 

Maybe an easier way to query the system and see what is available would be better so we don't have to be committed to updating these pages frequently.